x-pack/legacy/plugins/canvas/canvas_plugin_src/renderers/time_filter/components/datetime_range_absolute/datetime_range_absolute.tsx

```diff
@@ -36,9 +47,11 @@ export const DatetimeRangeAbsolute = ({ from, to, onSelect }) => (
         minDate={from}
         onValueChange={val => onSelect(from, val)}
         onSelect={val => {
+          if (!val || !to) return;
```

Comment:
This should be formatted with braces... curious why this wasn't done for you automatically.